### ISSUE TITLE
References to `get.delta.chat`

### ISSUE THREAD

**Issue by theAkito** [ArcaneChat/android#29]
https://github.com/search?q=repo%3AArcaneChat%2Fandroid%20get.delta.chat&type=code

I'm wondering, whether this is intentional or it was just not yet worked on.

There are plenty of references to `get.delta.chat` in the app (source code), as well as documentation.
However, `get.delta.chat` does not offer a download for this specific fork of Delta Chat.

Is this to be modified at some point or why are these links (still) there?

**Comment by adbenitez** [comment]
yeah, sadly updating that for every language will be quite some work and cause of merge conflicts, not sure I can update it properly for all languages, and if some part talks about desktop there is not even a desktop app for ArcaneChat at the moment

**Comment by theAkito** [comment]
> yeah, sadly updating that for every language will be quite some work and cause of merge conflicts, not sure I can update it properly for all languages, and if some part talks about desktop there is not even a desktop app for ArcaneChat at the moment

I thought, it would be a good first step to at the minimum change the link, so when people click on it, at least the correct destination is reached.

**Comment by adbenitez** [comment]
I replaced get.delta.chat with github.com/ArcaneChat, thanks for taking care and reporting
